src/main/kotlin/net/irisshaders/lilybot/extensions/util/ThreadControl.kt

```diff
@@ -37,15 +34,16 @@ class ThreadControl : Extension() {
 				description = "Rename a thread!"
 
 				check { isInThread() }
+				check { configPresent() }
 
 				action {
 					val threadChannel = channel.asChannel() as ThreadChannel
 					val member = user.asMember(guild!!.id)
 					val roles = member.roles.toList().map { it.id }
 
-					val moderatorRoleId = getConfigPublicResponse("moderatorsPing") ?: return@action
+					val config = DatabaseHelper.getConfig(guild!!.id)!!
 
-					if (moderatorRoleId in roles || threadChannel.ownerId == user.id) {
+					if (config.moderatorsPing in roles || threadChannel.ownerId == user.id) {
```

Comment:
config.moderatersPing is the ID, will this work correctly?

Reply:
Yes, we go ID to begin with, the old variable was `moderatorRoleId`